src/helpers/optionalStacking/common.ts

```diff
@@ -23,70 +20,44 @@ export function getViewByTwoAttributes(
     return [viewBy] as VisualizationObject.IVisualizationAttribute[];
 }
 
-export interface ISanitizedBucketsAndStackingConfig {
-    buckets: VisualizationObject.IBucket[];
-    config: IChartConfig;
+/**
+ * multiple measures => ignore computeRatio
+ */
+export function sanitizeMeasures(
```

Comment:
sanitizeMeasures seems to me too generic name. If it sanitizes only 'ComputeRatio' I would call it 'SanitizeComputeRatioOnMeasures'